python/ray/autoscaler/_private/resource_demand_scheduler.py

```diff
@@ -932,7 +933,9 @@ def get_bin_pack_residual(
 
 def _fits(node: ResourceDict, resources: ResourceDict) -> bool:
     for k, v in resources.items():
-        if v > node.get(k, 0.0):
+        if v > node.get(
```

Comment:
Do we have unit tests to cover implicit resources autoscaling? 

- Test we have max_relica_per_node = 1 & schedule 5 actors. Make sure 5 instances are started at the same time.